_docs/Qabel-Protocol-Box.md

```diff
@@ -101,15 +101,14 @@ Get a list of available prefixes
 ### Authentication
 The auth resource is only used by the block server and should only be exposed to internal servers (e.g. localhost)
 
-* Resource: /api/v0/auth/\<prefix\>/\<path\>
+* Resource: /api/v0/auth/
 * Method: GET|POST|DELETE
```

Comment:
The method is always POST.

Forgot to post this in the first round of comments…